.devcontainer/devcontainer.json

```diff
@@ -21,36 +21,36 @@
 		"terminal.integrated.shell.linux": "/bin/bash",
 		"clang-tidy.compilerArgs": [
 			// TODO: see if there is a way to use environment variables for workspace root (I tried)
-			"-I/workspaces/magma/build/c/oai/tasks/s1ap/r15/",  // Requires `make build_oai` to generate
```

Comment:
I should have commented on this in your Bazel PR - but now with your tooling I think all of this is useless and should be removed? It was an incomplete list already.  The downside will be that CMake clang-tidy won't work but it wasn't really functional anyway.

Reply:
ok will remove then :)